lib/rubocop/cop/lint/redundant_cop_disable_directive.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
-# The Lint/RedundantCopDisableDirective cop needs to be disabled so as
-# to be able to provide a (bad) example of a redundant disable.
-# rubocop:disable Lint/RedundantCopDisableDirective
 module RuboCop
   module Cop
+    # The Lint/RedundantCopDisableDirective cop needs to be disabled so as
+    # to be able to provide a (bad) example of a redundant disable.
+    # rubocop:disable Lint/RedundantCopDisableDirective
```

Comment:
Need to confirm that moving these types of comments doesn't result in accidentally generating unintentional documentation for the module defined below it (here `Lint`).

Reply:
I don't believe it should, I'm pretty sure that only comments directly on the class are used. You could run `rake update_cops_documentation` to check (but make sure you don't add adoc changes to this PR please).

Reply:
Alternately you could disable your cop for this file in `.rubocop.yml`.